build/codegen/codegen.sh

```diff
@@ -14,29 +14,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+##GROUP_VERSIONS="ceph.rook.io:v1"
+
+scriptdir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+basedir="$(cd "${scriptdir}/../.."  && pwd)"
+apisdir="${basedir}"/pkg/apis
+clientdir="${basedir}"/pkg/client
+
+boilerplate="${CODE_GENERATOR}/examples/hack/"boilerplate.go.txt
+kube_codegen="${CODE_GENERATOR}/kube_codegen.sh"
```

Comment:
Only used once in the script. No need to redirect to var for this.

Reply:
done. 😢